src/System.IO.Compression/tests/Utilities/ZipTestHelper.cs

```diff
@@ -0,0 +1,390 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Compression.Test
+{
+    public partial class ZipTest
+    {
+        #region filename helpers
+
+        public static String bad(String filename) { return Path.Combine("ZipTestData\\badzipfiles", filename); }
```

Comment:
Yes. I unfortunately spent quite some time cleaning these up in our other tests.